static/templates/modals/dummy-confirm.html

```diff
@@ -0,0 +1,12 @@
+<div class="modal-header">
```

Comment:
Do we need this file? Don't like having test files mixed up with real files and seems like one could test against static/templates/modals/confirm.html just as well